addon/components/markdown-to-html.js

```diff
@@ -26,14 +26,23 @@ export default Ember.Component.extend({
       'tasklists',
       'smoothLivePreview'
     );
+    var markdown = this.get('markdown') || '';
+    var source;
+
+    if (markdown instanceof Ember.Handlebars.SafeString) {
```

Comment:
I'm unsure this will work with Ember.String.safeString.  Perhaps it may?  Can you write a test to validate that.